PureBasicIDE/Issues.pb

```diff
@@ -377,9 +377,19 @@ Procedure Issues_CreateFunction(*Entry.ToolsPanelEntry, PanelItemID)
   AddGadgetColumn(#GADGET_Issues_List, 3, Language("Misc", "File"), IssuesCol4)
   AddGadgetColumn(#GADGET_Issues_List, 4, Language("ToolsPanel", "Priority"), IssuesCol5)
   
-  ButtonImageGadget(#GADGET_Issues_SingleFile, 0, 0, 0, 0, ImageID(#IMAGE_IssueSingleFile), #PB_Button_Toggle)
-  ButtonImageGadget(#GADGET_Issues_MultiFile,  0, 0, 0, 0, ImageID(#IMAGE_IssueMultiFile), #PB_Button_Toggle)
-  ButtonImageGadget(#GADGET_Issues_Export,     0, 0, 0, 0, ImageID(#IMAGE_IssueExport))
+  If EnableAccessibility
+    ; TODO: These should eventually be native optionGadgets. As it stands currently though, PB doesn't support changing the selection with the keyboard, so we stick with buttons for now.
+    ButtonGadget(#GADGET_Issues_SingleFile, 0, 0, 0, 0, Language("AccessibilityStuff", "SingleFile"), #PB_Button_Toggle)
```

Comment:
Hi,
I've now made the controls use the same language strings as are used for the tool tips; they're good enough for accessibility. I'm not sure on the resizing, I'm completely blind so am unable to test that. In a perfect world I'd just set an accessibility label on the controls, are you aware of a way to do this?

Reply:
> In a perfect world I'd just set an accessibility label on the controls, are you aware of a way to do this?

I actually just realized that you can call `SetGadgetText()` on a `ButtonImageGadget()` which won't be visible but both NVDA and Narrator will read the text when you select it. Would that do the trick?

If we want to use that in the IDE it probably makes sense to try to make this an official feature for the gadget so it would be cross-platform. I will look into that.